### DISCUSSION TITLE
有没有可能适配dhizuku呢

### DISCUSSION THREAD

**Discussion by txwfisher** [LSPosed/LSPatch#189] [Q&A]
如题

**Comment by Hakuin123** [comment]
Shizuku权限仅在静默安装修补后的应用时需要，不授予也可手动安装以正常使用，所以感觉优先级不高

（另外提建议的话应该发在issue里面）

**Comment by txwfisher** [comment]
好的，知道了，不太清楚，抱歉



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月17日(周二) 晚上6:34
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [LSPosed/LSPatch] 有没有可能适配dhizuku呢 (Discussion #189)




 
Shizuku权限仅在静默安装修补后的应用时需要，不授予也可手动安装以正常使用，所以感觉优先级不高
 
（另外提建议的话应该发在issue里面）
